catalog/task/prepare-skopeo-image-string/0.1/prepare-skopeo-image-string.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: prepare-skopeo-image-string
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to create a string from a snapshot that can be passed to skopeo
+  params:
+    - name: snapshot
+      type: string
+      description: The Snapshot in JSON format containing a single container image
+  results:
+    - name: skopeoImageString
+      description: |
+        The container image extracted from the snapshot, prepended by docker://
+  steps:
+    - name: apply-mapping
+      image:
+        quay.io/hacbs-release/release-base-image@sha256:9e7fd1a3ccf0d2c8077f565c78e50862a7cc4792d548b5c01c8b09077e6d23a7
+      script: |
+        #!/usr/bin/env sh
+        set -eux
+
+        echo "docker://"$(jq '.components[0].containerImage' <<< '$(params.snapshot)' | cut -d '"' -f 2) | \
```

Comment:
You can use `jq -r` for raw output without `""` and that way the `cut` is not needed.

Reply:
https://github.com/hacbs-release/release-bundles/pull/81

Reply:
thanks